src/main/java/uk/gov/companieshouse/controller/AppealController.java

```diff
@@ -60,8 +59,13 @@ public AppealController(AppealService appealService) {
             return ResponseEntity.status(HttpStatus.UNAUTHORIZED).build();
         }
 
+        if (!appealReasonValidator.isValid(appeal.getReason())) {
+            LOGGER.error("Appeal reason not valid with user id {} and penalty reference {}, {}",
+                userId, penaltyReference, appeal.getReason());
+            return ResponseEntity.status(HttpStatus.BAD_REQUEST).build();
```

Comment:
It could potentially be helpful to attach a reason that this was a bad request. Probably matching the validation response when an other field is invalid.

Reply:
Something like
`return ResponseEntity.status(HttpStatus.BAD_REQUEST).body("Not Other or Illness reason");`